src/DynamicMvvm.Uno/DynamicMvvm.Uno.csproj

```diff
@@ -35,9 +35,9 @@
 		<DefineConstants>$(DefineConstants);__WASM__</DefineConstants>
 	</PropertyGroup>
 
-	<ItemGroup Condition="'$(TargetFramework)'=='xamarinios10' or '$(TargetFramework)'=='monoandroid11.0' or '$(TargetFramework)'=='monoandroid12.0' or '$(TargetFramework)'=='netstandard2.0'">
-		<PackageReference Include="Uno.UI" Version="4.0.7" />
-		<PackageReference Include="Uno.SourceGenerationTasks" Version="4.1.0" />
+	<ItemGroup Condition="'$(TargetFramework)'=='xamarinios10' or '$(TargetFramework)'=='monoandroid12.0' or '$(TargetFramework)'=='monoandroid13.0' or '$(TargetFramework)'=='netstandard2.0'">
```

Comment:
My mistake, my tests on UnoApplicationTemplate aren't conclusive because that app targets the Uno.WinUI version of this library whereas we're updating the Uno.UI version, I'll have to retest